src/catalog/product-price-minimum-advertised.md

```diff
@@ -3,3 +3,15 @@ title: Minimum Advertised Price
 ---
 
 Merchants are sometimes prohibited from displaying a price that is lower than the manufacturer’s suggested retail price (MSRP). Minimum Advertised Price (MAP) gives you the ability to remain in compliance with the manufacturer’s requirements while offering your customers a better price. Because requirements differ from one manufacturer to another, you can configure your store to prevent the display of your actual price on pages where it is not allowed to appear according to the terms of the manufacturer.
+
+## How it works?
+
+The Minimum Advertised Price Magento extension adds a dedicated "Click for Price" button instead of the regular product price. Therefore, if the price in your store is below the minimum set price for that product, the price is either not displayed, or the list price or market price is displayed with a strikethrough to emphasize that your price is lower. When the buyer clicks the button "Click to find out the price" - only then the actual price at which you are selling the product becomes visible.
```

Comment:
```suggestion
The Minimum Advertised Price Magento extension adds a dedicated "Click for Price" button instead of the regular product price. Therefore, if the price in your store is below the minimum set price for that product, the price is either not displayed, or the list or market price is displayed with a strikethrough to emphasize that your price is lower. When the buyer clicks the button, "Click to find out the price" - only then will the actual price at which you are selling the product become visible.
```

Reply:
Done ✅